proto/c1/config/config.proto

```
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+import "c1/config/rules.proto";
+
+package c1.config.v1;
```

Buf (1.47.2)

Files with package "c1.config.v1" must be within a directory "c1/config/v1" relative to root but were in directory "c1/config".

(PACKAGE_DIRECTORY_MATCH)